src/pages/dashboard/components/header/Header.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from 'react'
+import { NavBar } from '../navbar/NavBar'
+import './Header.scss'
+
+function Header() {
+  const [display, setDisplay] = useState('')
+
+  function showAppBar() {
+    display === 'none' ? setDisplay('block') : setDisplay('none')
+  }
+
+  return (
+    <>
+      <div className="header">
+        <div className="header-container justify-between items-center">
+          <div
+            role="presentation"
+            className="app-bar-menu h-8 w-8 ml-2"
+            onClick={showAppBar}
+            onKeyDown={showAppBar}
+          >
+            <div className="app-bar"></div>
+            <div className="app-bar"></div>
+            <div className="app-bar"></div>
+          </div>
+          <div className="box-container flex justify-between items-center">
+            <div className="brand-banner">myStays.com</div>
+            <div className="flex justify-between items-center mr-7 ">
+              <svg
```

Comment:
svg image can be moved to a separate .svg file and import as an image here

Reply:
done